package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eslint-config-edx",
+  "version": "0.0.1",
+  "description": "ESLint config for edX JavaScript code.",
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bjacobel/eslint-config-edx.git"
+  },
+  "keywords": [
+    "eslint",
+    "eslintconfig",
+    "edx",
+    "mooc"
+  ],
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/edx/eslint-config-edx/issues"
```

Comment:
The URLs are a bit inconsistent here with some being "bjacobel/eslint-config-edx" and others being "edx/eslint-config-edx" rather.  It might be easier to be consistent so that a find and replace could catch them all.